libs/shared/methodologies/ai-attachment-validator/src/ai-attachment-validator.service.spec.ts

```
@@ -0,0 +1,299 @@
+        isValid: false,
+        validationResponse: VALIDATION_UNAVAILABLE_MESSAGE,
+      });
+    });
+
+    it('should correctly map the DTO for the API call', async () => {
+      const document = stubDocument();
+      const dto = stubAiValidateAttachmentDto({
+        document,
+      });
+
+      delete dto.additionalContext; // Ensure additionalContext is not included in the test
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)